### ISSUE TITLE
Live Reload in Development Environment

### ISSUE THREAD

**Issue by daemon1024** [kubearmor/KubeArmor#901]
Integrating Live Reloading with https://github.com/cosmtrek/air in our development environment. 

This will allow us to only make changes and KubeArmor would automatically rebuild and start once the changes are saved, not needing to execute make targets again and again.

**Hi, If it is the first time that you contribute to KubeArmor, follow these steps:**
Write a comment in this issue thread to let other possible contributors know that you are working on this bug. For eg : _Hey all, I would like to work on this issue._, checkout [Contributing Guide ](https://github.com/kubearmor/KubeArmor/blob/main/CONTRIBUTING.md)firesparkles and feel free to ask anything related to this issue in this thread or on our [slack channel](https://kubearmor.herokuapp.com/) ✌🏽

**Comment by 0xquark** [comment]
Hi, I would like to work on this issue.

**Comment by nyrahul** [comment]
Folks, looks like https://github.com/cosmtrek/air is a GPL3. While Apache2.0 is compatible with GPL3, it is required that the target software be released as GPL3. I am not sure if we should be integrating with air because of this reason. Any thoughts?

**Comment by VedRatan** [comment]
Yes @nyrahul , you're right, since Air is released under GPL 3.0 license our software should also be having GPL 3.0 license release, else we may have to update our license in order to integrate Air in our project.

**Comment by sarthaksarthak9** [comment]
@nyrahul is this issue is still up

**Comment by sarthaksarthak9** [comment]
I would love to solve it
